users/models.py

```diff
@@ -171,3 +173,10 @@ def email_user(self, subject, message, from_email=None):
         Sends an email to this user.
         """
         send_mail(subject, message, from_email, [self.email])
+
+
+@receiver(post_save, sender=CustomUser)
+def update_from_manager_to_employee(sender, **kwargs):
```

Comment:
This function also does not have tests. Please add them

Reply:
Done